ui/src/actions/literature.ts

```diff
@@ -302,30 +302,35 @@ export function setCurateDrawerVisibility(referenceId: number | null) {
 }
 
 export function curateReference({
-  recordId,
+  id,
+  recordUuid,
   revisionId,
   referenceId,
   newReferenceId,
 }: {
-  recordId: number;
+  id: number;
```

Comment:
isn't this the `record_id`? We are reading it from the router right?

Reply:
We need two parameters here -  _recordUuuid_ is uuid, and _id_ is control_number. Should I rename id to recordId?

Reply:
it will be more clear :)